openiddict/src/Audacia.Auth.OpenIddict/Token/PasswordClaimsPrincipalProvider.cs

```diff
@@ -66,7 +66,7 @@ public async Task<ClaimsPrincipal> GetPrincipalAsync(OpenIddictRequest openIddic
 
         // Validate the username/password parameters and ensure the account is not locked out.
         var userProxy = new UserWrapper<TUser, TId>(user);
-        var result = await _signInManager.CheckPasswordSignInAsync(user, openIddictRequest.Password, lockoutOnFailure: true).ConfigureAwait(false);
+        var result = await _signInManager.CheckPasswordSignInAsync(user, openIddictRequest.Password!, lockoutOnFailure: true).ConfigureAwait(false);
```

Comment:
Add this non-null assertion as the same is done above on the username - build failed without a null check